bundles/core/org.eclipse.smarthome.core.ephemeris/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0 http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="system:ephemeris">
+		<parameter name="weekendDays" type="text" required="true" multiple="true">
+			<default>Saturday,Sunday</default>
```

Comment:
Hm, I am actually not sure if such a "multiple" parameter can have defaults assigned like that... (@htreu Do you happen to know?).

Reply:
Confirmed, it's not the right way do define a multiple parameter default value :(